strings.txt

```diff
@@ -8784,11 +8784,11 @@ SETUP_NOROLEFILTER_DESC
 	PL	Filtrowanie może zostać włączone podczas przeglądania listy wykonawców, aby wyświetlić tylko te albumy i utwory, które są zgodne z wybraną rolą kontrybutora (np. ALBUMARTIST, COMPOSER itp.).
 	SV	När du bläddrar genom "Artister" kan du filtrera så att bara album och låtar som matchar den valda rollen ("ALBUMARTIST", "COMPOSER", etc.) visas.
 
-SETUP_COMPOSERINARTISTS
+SETUP_ROLESINARTISTS
```

Comment:
Oh, just spotted this one... hopefully Material (or other plugins) doesn't use this, or it'll be broken. Let's watch out for complaints... (didn't find any reference in a quick search)